src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -123,6 +123,7 @@ public class MSItems
 	public static final RegistryObject<Item> TOOTHRIPPER = REGISTER.register("toothripper", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 3, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(4)), new Item.Properties().defaultDurability(1200)));
 	public static final RegistryObject<Item> SHADOWRAZOR = REGISTER.register("shadowrazor", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 4, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(4)).add(OnHitEffect.onCrit(OnHitEffect.enemyPotionEffect(() -> new MobEffectInstance(MobEffects.POISON, 100, 0)))), new Item.Properties().defaultDurability(1500)));
 	public static final RegistryObject<Item> PRINCESS_PERIL = REGISTER.register("princess_peril", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 5, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(4)).add(OnHitEffect.onCrit(OnHitEffect.enemyPotionEffect(() -> new MobEffectInstance(MobEffects.POISON, 100, 0)))), new Item.Properties().defaultDurability(1650)));
+	public static final RegistryObject<Item> CHEESE_KNIFE = REGISTER.register("cheese_knife", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 0, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(3)), new Item.Properties()));
```

Comment:
Going off the grist cost generator sheet, the current cost factors in the "edible" property. Is that something you still intend on adding?

Reply:
Im undecided at the moment, I think ill delete this PR and just include this item in my cheese lands branch.

Reply:
Please do not! Its best to do PRs for individual components, even if they come together into one update. It reduces delays if one specific component does not pass review and it also makes the reviewing process cleaner

Reply:
Ah okay, in that case I will just leave this for now until I decide what to do with it.

Reply:
I would recommend just doing a recalculated version without edible (the universal cost should instead come out to 126)